docs/source/yelpsoa_configs.rst

```diff
@@ -609,6 +609,13 @@ Each Tron **action** of a job MAY specify the following:
     then the command should be something available in the docker container (it should NOT
     start with ``paasta local-run``).
 
+  * ``aws_credentials_yaml``: Path to the yaml file to load aws keys from. The yaml file must
+    have keys for aws_access_key_id and aws_secret_access_key. Please note that the file path
```

Comment:
How about telling the user what this will do, e.g. "Path to the yaml file containing credentials to be set in the task's AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables. If the file path does not exist, or the file does not contain keys for aws_access_key_id and aws_secret_access_key, those variables will be unset."

Reply:
Will they actually be unset, or set to ''?

Reply:
They will. The exception will happen before environment variable sets.